en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+* 1S - 3.7V
+* 2S - 7.4V
+* 3S - 11.1V
+* 4S - 14.8V
+* 5S - 18.5V
+* 6S - 22.2V
+
+This setting sets the number of cells connected in series PX4 should monitor.
+
+### Full cell voltage
+A chemical battery cells output voltage depends on many parameters like the way the battery was built, temperature, load, if the battery was damaged. But for a healthy battery under normal conditions with no load attached the remaining capacity is the main factor.
+Normal li-po batteries should be charged by dedicated charging devices such that each cell reaches a maximal voltage of exactly 4.2V. But after the battery was fully charged and is stored for later use the voltage starts to decrease even with no external load at all.
+That's why this setting should be set just a bit lower than the maximal voltage of a cell otherwise the fuel estimate will likely already be below 100% by the time you connect the battery to your autopilot.
+
+### Empty cell voltage
+Most battery cells particularly also li-pos suffer a lot when they get used after they are "empty". To say if they are empty is not binary and not accurately possible by only knowing the cell voltage.
+On one hand if a li-po ever sees a voltage lower than 3.2V even if it's under load it will suffer significant damage for sure. On the other hand the nominal cell voltage 3.7V without load is where the battery starts to get into the empty range and that's the most conservative voltage to consider it empty.
+In between these extremes the earlier you put the battery to charge again the longer the life span will be and the less capacity degrade you'll see.
+
+> **Hint** As my personal most general rule of thumb if the first li-po cell raw voltage gets below 3.5V during flight (under load) I start to land immediately.
+
+When speaking about the empty cell voltage we need to distinguish:
+* Raw voltage under load, the voltage directly measured while flying
+This value highly varies depending on the momentary current draw from the cells. As a result you will see fluctuating battery percentages
+* Rest voltage without a load when the battery is resting for some time
+Best suitable measurement to infer charge status of the cell.
+* Load compensated voltage calculated in air (under load)
+Ideally you want to have a load compensated voltage available during flight which reflects the rest voltage without load. but in practise the load compensated value will always be a bit below what you end up with because the battery "recovers" its unloaded voltage over time without gaining usable capacity.
+
+## Without load compensation
+With the default parameter set there is no load compensation enabled because if you don't have a current sensor, it might be uncalibrated or the internal resistance of your battery is completely off the resulting compensation might not make sense.
+
+If you don't want to use load compensation set the empty cell voltage to the value that should result in 0% battery estimation.
```

Comment:
I don't know what this means. Specifically "how would I guess the value that should result in 0% battery estimation"?

Reply:
Sorry I'm not super good at writing 😇 
If you have `BAT_V_EMPTY` = 3.5V, ...
- ... you don't use load compensation and the battery reads 3.5V per cell under load you see exactly 0%.
- ... you use load compensation and the battery reads 3.5V per cell under load and it gets compensated to 3.55V per cell you see maybe ~8%.

Therefore if you want to tune your load compensation well for the vehicle battery combination you use it's best to have a log with data to see what value gives the best results. You can also tune by trial and error.